R/aaa-auto.R

```diff
@@ -3580,25 +3580,6 @@ find_cycle_impl <- function(graph, mode=c("out", "in", "all", "total")) {
   res
 }
 
-simple_cycles_impl <- function(graph, mode=c("out", "in", "all", "total"), min.cycle.length=-1, max.cycle.length=-1) {
```

Comment:
Do you see a downside in keeping this and making `simple_cycles()` a wrapper of this function?

Reply:
Downsides are that the `mode <- ` lines must be repeated in the wrapper and we still need to add the `is.null()` checks. Overall, it won't be a simpler solution.

That said I have no strong preference.